src/pages/GetInvolved/Club.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { withRouter, Link } from "react-router-dom";
+import Layout from "components/Layout/Layout";
+import { ReactComponent as Website } from "../../assets/website.svg";
+import { ReactComponent as Facebook } from "../../assets/facebook.svg";
+import { ReactComponent as Instagram } from "../../assets/instagram.svg";
+import { ReactComponent as Twitter } from "../../assets/twitter.svg";
+
+const Club = ({ match }) => {
+  let clubDataAll = useSelector((state) => state.clubs);
+  let currentClub = match.params.clubId;
+  let clubData = clubDataAll.find((object) => object.slug === currentClub);
+
+  if (!clubData) {
+    return null;
+  } else {
+    let { club_description,club_image,club_contact,club_social_media } = clubData.acf.club;
+    let { title } = clubData.acf.seo;
```

Comment:
this is still the wrong title - SEO titles are different then page titles

`console.log(clubData)` and look for `title.rendered` that is the correct title